KotlinMMP/androidApp/src/main/java/and/jemison/kotlinmmp/android/MainActivity.kt

```diff
@@ -14,6 +15,16 @@ class MainActivity : AppCompatActivity() {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
 
+        val moodOptionsPage: ViewPager2 = findViewById(R.id.mood_options_view_pager)
+
+        val moodOptionFragments: ArrayList<Fragment> = arrayListOf(
+            MoodOptionFragment("Not So Great!", ContextCompat.getColor(applicationContext, R.color.red)),
+            MoodOptionFragment("Doing Great!", ContextCompat.getColor(applicationContext, R.color.yellow))
+        )
+
+        val adapter = MoodOptionsAdapter(moodOptionFragments, this)
+        moodOptionsPage.adapter = adapter
+
         if (BuildConfig.APP_CENTER_KEY_LOCAL != "noLocalKey") {
```

Comment:
App center linking should be the first thing that happens in the onCreate, or it will miss out registering any crashes that occur before that point. We could start splitting the logic into separate init functions to make this clearer.